hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java

```diff
@@ -186,6 +187,16 @@ public String getString(String columnLabel) {
 
 	@Override
 	public boolean getBoolean(String columnLabel) {
```

Comment:
Assuming that this change is correct, it should also be applied to `getBoolean(int colPos)`.
It's probably safer to collect the logic in a private method

Reply:
good suggestion

Reply:
... but not sure how to set up a test to test `colPos`

Reply:
Let's worry first about the tests that we have and are failing.

But I like how you think  :-)